src/main/java/com/google/sps/servlets/AddNewTabServlet.java

```diff
@@ -0,0 +1,69 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Key; 
+import com.google.appengine.api.datastore.KeyFactory; 
+import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import com.google.sps.data.Constants;
+import com.google.sps.data.DatastoreManager;
+import com.google.sps.data.Post;
+import com.google.sps.data.Representative;
+import com.google.sps.data.DatastoreManager;
+import java.io.IOException;
+import java.net.URLEncoder; 
+import java.util.Arrays;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.ServletException;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+/** 
+InsertRepDatastoreServlet currently inserts the representative Donald
+Trump into the datastore as a hard code for the MVP.
+*/
+
+@WebServlet ("/add_new_tab")
+public class AddNewTabServlet extends HttpServlet {
+    private static final Logger logger = LogManager.getLogger("AddNewTabServlet");
+    private static final String NEW_TAB_NAME = "tabName";
```

Comment:
nit (since i don't think I mentioned it for other PRs): A better name for this constant and others is NEW_TAB_NAME_PARAM (_KEY, _COLUMN,  _EXTRA or whatever makes sense in context).

The suffix makes it clear that this isn't really the actual name of the tab, but the name of the parameter that contains the tab name information.